### ISSUE TITLE
Background processes broken for bash

### ISSUE THREAD

**Issue by Twiggeh** [microsoft/vscode-js-debug#512]
Can't create background processes from `tasks.json` . Basically if I append `&` to a command it doesn't work …
## tasks.json
```js
  "tasks": [
    {
      "type": "shell",
      "label": "start webpack",
      "command": "cd bin && ./startWebpack.sh &"
    },
]
```

## startWebpack.sh
```bash
#!/bin/bash
echo "Starting webpack-dev-server"
cd .. && cd client && yarn debug &
```

## Example recording

I start the command just by itself in bash & the script runs, if I start it from vsc it doesn't. This setup still worked yesterday …
![Peek 2020-06-08 15-46](https://user-images.githubusercontent.com/23149166/84037905-86855280-a99f-11ea-8fbd-c2c200466c38.gif)

**Comment by connor4312** [comment]
I don't think this is related to js-debug, as tasks are handled separately. Transferring to vscode and Alex.
